README.rst

```diff
@@ -36,6 +36,11 @@ To install pytest-variables::
 
   pip install pytest-variables
 
+You can also install optional support for `HJSON <http://hjson.org>`_ and `YAML <http://pyyaml.org>`_ files::
+
+  pip install "pytest-variables[hjson]"
+  pip install "pytest-variables[yaml]"
+
```

Comment:
Let's create a subheading for 'Additional formats' and list the ahead of showing an example of how to install them.